spec/models/institution_builder_spec.rb

```diff
@@ -948,13 +949,5 @@ def check_ipeds_ic_ay
         expect(institutions.find_by(facility_code: '1VZZZZZZ').approved).to be_truthy
       end
     end
-
-    describe 'when generating versioned school certifying official table' do
```

Comment:
should this test be rewritten to account for the new behavior of `build_school_certifying_officials` instead of being removed?